### PR TITLE
add trait promotion without array

### PR DESCRIPTION
**Pull request by alex-s-gardner**
This address #157

### CODE REVIEW

**Comment by rafaqz** [comment]
Perfect, just needs a test

**Review by rafaqz — APPROVED** [review]

File: src/wrappers.jl [hunk 1]
```diff
@@ -425,6 +432,9 @@ end
 function FeatureCollection(fc::FeatureCollection; crs=crs(fc), extent=extent(fc))
     FeatureCollection(parent(fc), crs, extent)
 end
+function FeatureCollection(fc::Feature; crs=crs(fc), extent=extent(fc))
```
Comment (rafaqz) [inline src/wrappers.jl:435]: Actually this isn't quite right. Instead you want to check `isfeature` in the method below

**Review by rafaqz — CHANGES_REQUESTED** [review]

File: src/wrappers.jl [hunk 2]
```diff
@@ -425,6 +432,9 @@ end
...
+    FeatureCollection([fc], crs, extent)
+end
```
Comment (asinghvi17) [inline src/wrappers.jl:437]:
```suggestion
```

File: src/wrappers.jl [hunk 3]
```diff
@@ -428,6 +435,15 @@ end
...
     if isfeaturecollection(parent)
         FeatureCollection(parent, crs, extent)
+    elseif isfeature(parent)
+        # If `parent` is a single feature, wrap it in a featurecollection
+        # Note the use of `(@__MODULE__).crs(parent)` to explicitly reference
+        # `GeoInterface.crs`, since `crs` and `extent` are also keyword arguments.
+        FeatureCollection(
+            [parent], 
+            isnothing(crs) ? (@__MODULE__).crs(parent) : crs, 
+            isnothing(extent) ? (@__MODULE__).extent(parent) : extent
```
Comment (rafaqz) [inline src/wrappers.jl:445]:
```suggestion
        FeatureCollection(
            [parent], 
            isnothing(crs) ? Wrappers.crs(parent) : crs, 
            isnothing(extent) ? Wrappers.extent(parent) : extent
```

Comment (asinghvi17) [reply src/wrappers.jl]: Ah that's a lot simpler

**Review by rafaqz — APPROVED** [review]

**Comment by asinghvi17** [comment]
Should we bump and release?

**Pull request merged**